lib/src/cli.rs

```diff
@@ -1071,7 +1074,7 @@ async fn run_from_opt(opt: Opt) -> Result<()> {
                 late_dir: _,
             } => {
                 let unit_dir = &Dir::open_ambient_dir(normal_dir, cap_std::ambient_authority())?;
-                crate::generator::generator(root, unit_dir)
+                crate::generator::generator(&root, unit_dir)
```

Comment:
I think this will actually break things on the booted system because `etc` is a bind mount right?

Reply:
Ya.  Good point.  I wasn't thinking about the booted system.

I might de-factor this variable then.  It only has 2 users (if you count the two "internals" commands as one), and they need it for different things...